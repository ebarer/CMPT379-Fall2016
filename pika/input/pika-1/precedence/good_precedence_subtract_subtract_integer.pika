exec {

  print 68 - 34 - 79, _n_.
  print 67 - 47 - 4, _n_.
  print 90 - 45 - 38, _n_.
  print 15 - 23 - 91, _n_.
  print 92 - 28 - 9, _n_.
  print 28 - 16 - 39, _n_.
  print 19 - 65 - 53, _n_.
  print 37 - 93 - 48, _n_.
  print 82 - 93 - 72, _n_.
  print 3 - 63 - 82, _n_.
  print 98 - 60 - 23, _n_.
  print 35 - 20 - 88, _n_.

  print 65 - 93 - 24, _n_.
  print 75 - 8 - 28, _n_.
  print 92 - 18 - 57, _n_.
  print 36 - 95 - 47, _n_.
  print 54 - 86 - 40, _n_.
  print 93 - 10 - 19, _n_.
  print 34 - 85 - 88, _n_.
  print 15 - 85 - 77, _n_.
  print 73 - 92 - 35, _n_.
  print 37 - 21 - 34, _n_.
  print 43 - 98 - 4, _n_.
  print 24 - 68 - 85, _n_.

  print 68 - 40 - 21 - 27, _n_.
  print 91 - 49 - 11 - 70, _n_.
  print 51 - 62 - 8 - 36, _n_.
  print 95 - 21 - 59 - 22, _n_.
  print 45 - 53 - 38 - 53, _n_.
  print 72 - 80 - 52 - 54, _n_.
  print 22 - 96 - 94 - 62, _n_.
  print 85 - 69 - 89 - 33, _n_.
  print 78 - 55 - 60 - 35, _n_.
  print 88 - 13 - 28 - 42, _n_.
  print 79 - 15 - 15 - 27, _n_.
  print 29 - 1 - 15 - 25, _n_.

  print 80 - 4 - 52 - 92, _n_.
  print 64 - 88 - 32 - 97, _n_.
  print 54 - 28 - 77 - 54, _n_.
  print 8 - 22 - 69 - 38, _n_.
  print 1 - 41 - 28 - 94, _n_.
  print 61 - 46 - 8 - 10, _n_.
  print 94 - 15 - 11 - 31, _n_.
  print 21 - 20 - 82 - 99, _n_.
  print 14 - 61 - 69 - 65, _n_.
  print 92 - 7 - 18 - 12, _n_.
  print 57 - 4 - 76 - 34, _n_.
  print 21 - 84 - 4 - 68, _n_.

}

