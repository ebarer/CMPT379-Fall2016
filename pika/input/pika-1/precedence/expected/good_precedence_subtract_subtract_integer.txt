-45
16
7
-99
55
-27
-99
-104
-83
-142
15
-73
-52
39
17
-106
-72
64
-139
-147
-54
-18
-59
-129
-20
-39
-55
-7
-99
-114
-230
-106
-72
5
22
-12
-68
-153
-105
-121
-162
-3
37
-180
-181
55
-57
-135
